examples/2d/elixir_eulermulti_shock_bubble_3comp.jl

```diff
@@ -0,0 +1,70 @@
+using OrdinaryDiffEq
```

Comment:
You're right, the abbreviations might be misleading. I will change it to `3components`. 

For the shock_bubble test case I would rather not add `_2components` since it is typically a two component setup. 

There is still a question if elixirs which can use a variable number of components should have something like `_xcomponents` or similar in their name. This way we could make it clear that the user is allowed to use as many components as desired.

Reply:
I would not add `_xcomponents` unless it is the prominent "feature" of a particular elixir. That is, if you have configured an elixir that makes sense for an arbitrary number of components, and which you specifically added to demonstrate this capability, then yes. Otherwise, I would omit it.

Reply:
done in 657c2b2